### PR TITLE
Swerve2022: Added in subsystems and PhoenixUtil files

### PR DESCRIPTION
**Pull request by reevar24**

### CODE REVIEW

File: Swerve2022/src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -51,9 +53,170 @@
...
     public static final double kWheelDiaMeters        = Units.inchesToMeters(4.0); // 4in (39.37 in/meter)
     public static final double kGearRatio             = 6.75;
     public static final double kEncoderMetersPerCount = (kWheelDiaMeters * Math.PI) / (Falcon500.kEncoderCPR) / kGearRatio;
+    public static final double kTrackWidthMeters      = 0.6477;        // Measured track width
+
+    // Joystick tuning
+    public static final double kDriveXScaling         = 1.0;           // 1.0 is no scaling
+    public static final double kDriveYScaling         = 1.0;           // 1.0 is no scaling
+    public static final double kQuickTurnScaling      = 0.5;           // Scale by 50% of full speed
+    public static final double kSlowClimbScaling      = 0.3;           // Scale by 30% of full speed
```
Comment (hpark-24) [inline Swerve2022/src/main/java/frc/robot/Constants.java:66]: Is this related to Climber? If so, should we include it in the Constants for Swerve?

**Review by hpark-24 — APPROVED** [review]
Looks good!

**Review by jmullins16 — APPROVED** [review]
Looks good. If it all compiles cleanly, it can be merged.
